### PR TITLE
Update Alerting Docs

### PR DESCRIPTION
**Pull request by rekt0x**
Remove the outdated Telegram monitoring bot and replace it with Vordr, as discussed in https://gitlab.com/thorchain/devops/node-launcher/-/merge_requests/691.

### CODE REVIEW

File: thornodes/alerting.md
Comment (Orion-9R) [inline thornodes/alerting.md:24]: My 2 sats - this is fine as long as it is very clear that this is 3rd party software outside of the scope of THORSec.

Comment (rekt0x) [reply thornodes/alerting.md]: I added an extra disclamer that said that.

**Comment by Orion-9R** [comment]
LGTM - I don't have approval access.

@GrassRootsCrypto - can you take a look and approve + merge if good to go?

**Review by GrassRootsCrypto — APPROVED** [review]
Looks good @Orion-9R

**Comment by GrassRootsCrypto** [comment]
@Orion-9R - I don't seem to have rights to merge.

**Pull request merged**
